### ISSUE TITLE
Link the latest release to the direct latest download URL

### ISSUE THREAD

**Issue by vinibali** [go-gitea/gitea#31408]
### Feature Description

Hello there!
Could you please link the latest release to the latest download tag?
This would help to pick up the latest binaries on the go. The URL which would work in case of amd64 should be:
```
https://github.com/go-gitea/gitea/releases/latest/download/gitea-linux-amd64
```
but it's not pointing anywhere.
BR

**Comment by wxiaoguang** [comment]
The latest is set. But the problem is that the binary file itself contains the version number. So `https://github.com/go-gitea/gitea/releases/latest/download/gitea-linux-amd64` is redirected to `https://github.com/go-gitea/gitea/releases/download/v1.22.0/gitea-linux-amd64` but `gitea-linux-amd64` doesn't exist.

@techknowlogick

**Comment by bilogic** [comment]
@wxiaoguang i think the OP is asking for a permalink to the latest binary, possible?

**Comment by bilogic** [comment]
@vinibali https://github.com/go-gitea/gitea/releases/download/latest/gitea-1.25.5-linux-amd64 this works

**Comment by wxiaoguang** [comment]
> i think the OP is asking for a permalink to the latest binary, possible?

The example is above: https://github.com/go-gitea/gitea/issues/31408#issuecomment-2175176702

Users would like to download the latest Gitea binary by `https://some-host/some-dir/gitea-latest-amd64` without knowing the "version"

also cc @techknowlogick

**Comment by bilogic** [comment]
> Users would like to download the latest Gitea binary by `https://some-host/some-dir/gitea-latest-amd64` without knowing the "version"

1. Yes, same interpretation, but I don't think https://github.com/go-gitea/gitea/issues/31408#issuecomment-2175176702 addresses the problem 
2. The problem is that gitea releases have the version number embedded in their filenames, and github isn't able to replace them in the URL
3. If all releases are named `gitea-<os>-<arch>`, then the permalink can be https://github.com/go-gitea/gitea/releases/download/latest/gitea-linux-amd64

**Comment by wxiaoguang** [comment]
> 1. Yes, same interpretation, but I don't think [Link the latest release to the direct latest download URL #31408 (comment)](https://github.com/go-gitea/gitea/issues/31408#issuecomment-2175176702) addresses the problem

But I was just explaining the problem, I never said "it addresses the problem"

> 2. ... & 3. ....

Yes, the problem is also explained in https://github.com/go-gitea/gitea/issues/31408#issuecomment-2175176702 , " But the problem is that the binary file itself contains the version number."

Any new information besides my explanation?

**Comment by bilogic** [comment]
> Any new information besides my explanation?

No. Ok sorry, for some reason, I missed the part where you explained it. I guess because a solution was not described explicitly.

i.e. solution is to remove the version number from the filename
